### PR TITLE
Fix a few warnings

### PR DESCRIPTION
**Pull request by jrudolph**
- akka.stream.impl.ConstantFun => akka.util.ConstantFun
 - ''' => '\''
 - Left.a => Left.value
 - Remove unused imports

### CODE REVIEW

**Comment by jrudolph** [comment]
PLS BUILD

**Comment by jrudolph** [comment]
Something broke but I'm not yet sure what. Will investigate later.

**Comment by jrudolph** [comment]
Observed problem was #1786.

**Comment by jrudolph** [comment]
Removed the problematic change..

**Comment by jrudolph** [comment]
Missed one change that should rather be fixed in https://github.com/akka/akka-http/pull/1787/files.

File: akka-http-core/src/main/scala/akka/http/impl/engine/client/OutgoingConnectionBlueprint.scala [hunk 1]
```diff
@@ -7,7 +7,7 @@ package akka.http.impl.engine.client
 import akka.NotUsed
 import akka.annotation.InternalApi
 import akka.http.scaladsl.settings.{ ClientConnectionSettings, ParserSettings }
-import akka.stream.impl.ConstantFun
+import akka.util.ConstantFun
```
Comment (ktoso) [inline akka-http-core/src/main/scala/akka/http/impl/engine/client/OutgoingConnectionBlueprint.scala]: good :)

**Pull request merged**